### PR TITLE
hw1, Daria Suvorova

### PR DESCRIPTION
**Pull request by daaria-s**

### CODE REVIEW

File: src/test/java/hw1/Task1Test.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class Task1Test {
+    @Test
+    void validTest() {
+        assertEquals(Task1.minutesToSeconds("01:00"), 60);
```
Comment (vasilmik) [inline src/test/java/hw1/Task1Test.java:12]: поменяйте аргументы местами

File: src/test/java/hw1/Task6Test.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package hw1;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Task6Test {
+    @Test
+    void validTest() {
```
Comment (vasilmik) [inline src/test/java/hw1/Task6Test.java:9]: Рассмотрите corner case

Comment (daaria-s) [reply src/test/java/hw1/Task6Test.java:9]: что значит corner case?
